Dockerfile

```diff
@@ -1,5 +1,25 @@
-FROM google/cloud-sdk:latest
+FROM debian:jessie
 
-WORKDIR /app
+# Copied from https://hub.docker.com/r/google/cloud-sdk/~/dockerfile/
+# If there are upstrem changes copy them here
 
-COPY authenticate_and_run.sh ./
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y -qq --no-install-recommends wget unzip python php5-mysql php5-cli php5-cgi openjdk-7-jre-headless openssh-client python-openssl curl && apt-get clean
```

Comment:
Can we extend the Google Cloud Container if we copy parts of it? (If you think no, simply say so :D)